app/models/sitemap.rb

```diff
@@ -0,0 +1,13 @@
+class Sitemap < ActiveRecord::Base
+  include Fetchable
+
+  attr_readonly :url
+
+  belongs_to :searchgov_domain
+
+  validates :url, presence: true, uniqueness: true,
+   length: {maximum: 2000}, format: {with: /\Ahttps?:\/\/(\w+\.)?\w+\.gov(\/\w+)*(\/|\.\w+)?\z/, message: 'invalid url'}
+  validates :last_crawl_status, length: {maximum: 255}
```

Comment:
let's move this validation to `Fetchable`, and make sure there's a corresponding spec in `a record that behaves like a fetchable URL`